Tools/TestWebKitAPI/Tests/WebKit/AGXCompilerService.mm

```diff
@@ -25,7 +25,7 @@
 
 #import "config.h"
 
-#if WK_HAVE_C_SPI && PLATFORM(IOS)
+#if WK_HAVE_C_SPI && PLATFORM(IOS) || PLATFORM(VISION)
```

Comment:
`WK_HAVE_C_SPI && (PLATFORM(IOS) || PLATFORM(VISION))`, for correctness?

Reply:
good call!